### PR TITLE
Remove {{page}} macro from Glossaire files

### PR DESCRIPTION
**Pull request by cw118**
Part of #3886

### CODE REVIEW

File: files/fr/glossary/denial_of_service/index.md [hunk 1]
```diff
@@ -1,13 +1,9 @@
 ---
 title: Déni de Service
 slug: Glossary/Denial_of_Service
-tags:
-  - Attaque
-  - Déni de Service
-  - Glossaire
-  - Intro
-  - Sécurité
```
Comment (JuanVqz) [inline files/fr/glossary/denial_of_service/index.md:9]: Hi @cw118 should we avoid adding tags? I know you told me that they are optional for us, which one is the best approach?

Comment (cw118) [reply files/fr/glossary/denial_of_service/index.md:9]: If I remember correctly tags don't do much for translated-content, but I'm not the best person to ask — @SphinxKnight is it just for `fr` or should all locales avoid adding tags?

Comment (SphinxKnight) [reply files/fr/glossary/denial_of_service/index.md:9]: @JuanVqz hello, tags for localized pages are mostly a relicate of the old platform (kuma) and are not used anywhere or presented to the reader in any form. As such, I'd advise to remove/avoid them. That being written, they're not harmful so it's not a really big deal if they're kept.

Comment (SphinxKnight) [reply files/fr/glossary/denial_of_service/index.md:9]: To be completely honest, some tags may be used by some macros (cf. https://github.com/mdn/yari/blob/main/kumascript/macros/HTMLRef.ejs#L34 ) but this is not consistent depending on the section (in other words, not all sidebars (ex. JSRef or HTTPSidebar) use them)

Comment (SphinxKnight) [reply files/fr/glossary/denial_of_service/index.md:9]: On further inspection, 
1. A localized page shoud automatically/implicitly get the tags of its "parent" (i.e. English) page: https://github.com/mdn/yari/blob/main/kumascript/index.js#L67 (this also applies for other info in the frontmatter)
2. Here are the macros which are still used in English and which are using tags:

- `APIListAlpha` (used only on Web/API and targeting subpages with `Non-standard`, `Obsolete`, `Experimental`, `junk` )
- `HTMLRef` (used for many element pages, tag must have the "HTML XYZ" form)
  - apart from this form: current calls on en-US only relies on `Sectioning Root Element`, `multimedia`, `Deprecated`, `Web Components`, `Obsolete`
- `HTMLRefTable` (used only on Web/HTML/Element but scanning subpages)
- ~`LearnBox` which uses a parameter on tags to filter / there are only two calls for this macro, none of them uses this parameter~
- `ListGroups` used on Web/API and Web/Guide/API: only uses `Non-standard`, `Experimental` or `Obsolete` tags
- `MathMLRef` which only checks the tag `MathML:Element`
- `Non-standard*` which only uses the `Firefox OS` tag (present only on 9 pages in en-US)
- - `SVGRef` (used for many element pages, tag must have the "SVG XYZ" form)

So:
- Tags are not strictly necessary on localized pages
- Even for English pages, only a small subset of tags are still useful.

File: files/fr/glossary/denial_of_service/index.md [hunk 2]
```diff
@@ -1,13 +1,9 @@
 ---
 title: Déni de Service
```
Comment (SphinxKnight) [inline files/fr/glossary/denial_of_service/index.md:2]: A single/minor nitpicking for this PR  : ) 
```suggestion
title: Déni de service
```

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
one case nitpicking to go before merging :)

**Comment by SphinxKnight** [comment]
Thanks @cw118 !

File: files/fr/glossary/denial_of_service/index.md (hunk 1 above)
Comment (JuanVqz) [reply files/fr/glossary/denial_of_service/index.md:9]: @SphinxKnight @cw118 Thank you for your time and great explanation. :bookmark_tabs: 
we will remove the tags, let's do it...

**Review by SphinxKnight — APPROVED** [review]
🙏

**Pull request merged**